### PR TITLE
zh: update dm-worker config doc for 2.0

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed, added or deleted? (Required)
update dm-worker-config-file.md to `2.0`

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
preview: https://github.com/GMHDBJD/docs-dm/blob/dm-worker-config-file-2.0/zh/dm-worker-configuration-file.md

### CODE REVIEW

File: zh/dm-worker-configuration-file.md [hunk 1]
```diff
@@ -46,21 +31,12 @@ port = 3306
 
 | 配置项        | 说明                                    |
 | :------------ | :--------------------------------------- |
+| `log-level`   | 日志级别：debug, info, warn, error, fatal。默认为 info。   |
```
Comment (toutdesuite) [inline zh/dm-worker-configuration-file.md:34]:
```suggestion
| `log-level` | 日志级别：debug，info，warn，error，fatal。默认为 info。   |
```

File: zh/dm-worker-configuration-file.md [hunk 2]
```diff
@@ -46,21 +31,12 @@ port = 3306
...
 | `log-file`   | 日志文件，如果不配置日志会输出到标准输出中。   |
+| `name`   | 标识一个worker。   |
```
Comment (toutdesuite) [inline zh/dm-worker-configuration-file.md:36]:
```suggestion
| `name`   | 标识一个 DM-worker。   |
```

File: zh/dm-worker-configuration-file.md [hunk 3]
```diff
@@ -46,21 +31,12 @@ port = 3306
...
-### 数据库链接配置（from 配置项）
-
-| 配置项        | 说明                                    |
-| :------------ | :--------------------------------------- |
-| `host` | 上游数据库的 host。|
-| `port` | 上游数据库的端口。 |
-| `user` | 连接数据库使用的用户名。 |
-| `password` | 连接数据库使用的密码。注意：需要使用 dmctl 加密后的密码。 |
+| `advertise-addr` | DM-worker 向外界宣告的地址。 |
+| `join` | 对应一个或多个 DM-master 配置中的 `master-addr`。 |
```
Comment (yikeke) [inline zh/dm-worker-configuration-file.md:39]:
```suggestion
| `join` | 对应一个或多个 DM-master 配置中的 [`master-addr`](dm-master-configuration-file.md#global-配置)。 |
```
给个超链接方便用户直接查看提到的内容

File: zh/dm-worker-configuration-file.md [hunk 4]
```diff
@@ -15,29 +15,14 @@ category: reference
 # Worker Configuration.
 
 # Log configuration.
+log-level = "info"
 log-file = "dm-worker.log"
 
 # DM-worker listen address.
+name = "worker1"
```
Comment (WangXiangUSTC) [inline zh/dm-worker-configuration-file.md:22]: name 需要移一下位置，因为上面的 `listen address` 的注释是下面配置 worker-addr 的

File: zh/dm-worker-configuration-file.md [hunk 5]
```diff
@@ -14,30 +14,16 @@ category: reference
 ```toml
 # Worker Configuration.
```
Comment (toutdesuite) [inline zh/dm-worker-configuration-file.md:16]:
```suggestion
```

**Comment by GMHDBJD** [comment]
I remove dm-worker-configuration-file-full.md and remove some information in dm-worker-configuration-file.md because it is no longer needed in 2.0
@toutdesuite

**Comment by toutdesuite** [comment]
@GMHDBJD Review this PR again. LGTM. Note that if any other files link to `zh/dm-worker-configuration-file-full.md`, remember to remove the link.

**Comment by TomShawn** [comment]
@toutdesuite Please create a PR to add the alias `['/docs-cn/tidb-data-migration/dev/dm-worker-configuration-file-full/']` to the metadata section of `dm-worker-configuration-file.md`.

**Pull request merged**